src/backend/dx11/src/conv.rs

```diff
@@ -226,6 +226,15 @@ pub fn map_viewport(viewport: &Viewport) -> D3D11_VIEWPORT {
     }
 }
 
+pub fn map_scissor(rect: &Rect) -> D3D11_RECT {
+    D3D11_RECT {
+        left: rect.x as _,
+        top: rect.y as _,
+        right: (rect.x + rect.w) as _,
```

Comment:
wait a sec, why are we keeping `map_rect` incorrect instead of fixing and reusing it?